_locales/en/messages.json

```diff
@@ -646,6 +646,12 @@
     "configurationBatteryScale": {
         "message": "Voltage Scale"
     },
+    "configurationVbatPidCompensation": {
+        "message": "Vbat PID Compensation"
+    },
+    "configurationVbatPidCompensationHelp": {
+        "message": "Increase the PID values to compensate when Vbat gets lower. This will give more constant flight characteristics throughout the flight."
```

Comment:
Its in the mixer.c! Thats where it is being used.
Op 13 jul. 2016 01:10 schreef "Michael Keller" <notifications@github.com>:

> In _locales/en/messages.json
> <https://github.com/betaflight/betaflight-configurator/pull/111#discussion_r70539275>
> :
>
> > @@ -646,6 +646,12 @@
> >      "configurationBatteryScale": {
> >          "message": "Voltage Scale"
> >      },
> > +    "configurationVbatPidCompensation": {
> > +        "message": "Vbat PID Compensation"
> > +    },
> > +    "configurationVbatPidCompensationHelp": {
> > +        "message": "Increase the PID values to compensate when Vbat gets lower. This will give more constant flight characteristics throughout the flight."
>
> I don't want to be the spoilsport, but I also want to make sure I am not
> seeing things. The only reference to calculateBatteryPercentage that I
> could find is in the display code:
>
> ~/git/betaflight$ find . -name .git -o -name obj -prune -o -type f -exec grep calculateBatteryPercentage {} \; -print
> uint8_t calculateBatteryPercentage(void)
> ./src/main/sensors/battery.c
> uint8_t calculateBatteryPercentage(void);
> ./src/main/sensors/battery.h
>         uint8_t batteryPercentage = calculateBatteryPercentage();
> ./src/main/io/display.c
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/betaflight/betaflight-configurator/pull/111/files/c63af70999b71270e4c2c6b4d60295356c402a65#r70539275>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKQlhN3qydok8K75O5_GrjQK8WeUOhMdks5qVB7TgaJpZM4JKUzM>
> .
>